src/Api/Command/PayByBlikHandler.php

```diff
@@ -19,7 +18,7 @@ public function __invoke(PayByBlik $command): PayResult
         $this->setTransactionData($payment, $command->blikToken);
         $this->createTransaction($payment);
 
-        return $this->createResultFrom($payment);
+        return $this->createResultFrom($payment, false);
```

Comment:
```suggestion
        return $this->createResultFrom($payment, isRedirectBased: false);
```

It's not necessary, but makes it clearer;